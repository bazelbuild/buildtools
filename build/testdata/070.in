manifest = [
    # buildifier: table
                       ("", "", ""), ("ab", "bc", "cd"),("a", "b", "c")]
