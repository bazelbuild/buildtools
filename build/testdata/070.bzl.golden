foo(
    name = "foo",
    srcs = ["bbb", "aaa"] + CONSTANT + ["//ddd:ddd", "//ccc:ccc"],
    deps = select({
        "bar": ["//ggg:ggg", "//eee:eee"],
    }) + ["hhh", "fff"],
    data = ["jjj", "iii"] + bar(["not", "sortable", "arguments"]) + non_select({
        "baz": ["not", "sortable", "arguments"],
    }),
)
